src/tree_schema.h

```diff
@@ -1947,12 +1905,22 @@ LY_ERR lysc_iffeature_value(const struct lysc_iffeature *iff);
 /**
  * @brief Get the current status of the provided feature.
  *
- * @param[in] feature Compiled feature statement to examine.
+ * @param[in] feature Feature statement to examine.
  * @return LY_SUCCESS if feature is enabled,
  * @return LY_ENOT if feature is disabled,
  * @return LY_ERR in case of error (invalid argument)
  */
-LY_ERR lysc_feature_value(const struct lysc_feature *feature);
+LY_ERR lysp_feature_value(const struct lysp_feature *feature);
```

Comment:
Not sure if it is really needed when the value is now clearly set by `LYS_FENABLED` without anything else.

Reply:
Removed.